doc/sphinx/tutorial2.md

```diff
@@ -0,0 +1,81 @@
+---
+discourse: 4911,5164,5603,6756,8037
+---
+
+# Tutorial 2
+This tutorial will guide you through using Mir demos to see what Mir is capable
+of, and the usage of Mir libraries to build your own Wayland compositor.
+
+## Mir Demos
+Mir comes with a set of demos in its [example
+directory](https://github.com/canonical/mir/tree/main/examples) to showcase its
+abilities. This section will cover the installation and usage of these demos.
+
+
+### Installation
+Mir demos are available on Ubuntu as prebuilt binaries. For distros that don't
+have prebuilt binaries, examples can be built from source.
+
+<details>
+<summary> Installing Mir Demos on Ubuntu </summary>
+
+You can run the following command to install Mir demos and the drivers they
+use.
+```sh
+sudo apt install mir-demos mir-graphics-drivers-desktop
+```
```

Comment:
This works across all the Debian derived distros, no need to call out Ubuntu. I think we can also mention the `dnf` and `apk` incantations.

Reply:
I haven't actually used `dnf` or `apk` before but a quick grok at their docs show that the commands are:
1. `dnf install <package name>`
2. `apk add <package name>`

Would appreciate if you can confirm if these are correct.

Both [Fedora Packages](https://packages.fedoraproject.org/) and [Alpine Packages](https://pkgs.alpinelinux.org/packages) seem to find `mir-demos` but not `mir-graphics-drivers-desktop`. Are these drivers not needed on those distros?

Reply:
I don't think either has broken the drivers down the way we have. I think there's one big package.

E.g. https://pkgs.alpinelinux.org/contents?branch=edge&name=mir&arch=x86_64&repo=testing

shows the `server-platform`s

Reply:
The needed packages seem to be `mir-demos` and `mir` on both `dnf` and `apk`. Feel free to unresolve if I forgot anything (once I push my commits :))